web/themes/joinup/prototype/sass/components/_sidebar-menu.sass

```diff
@@ -61,7 +61,7 @@
 // Modifiers
 // --------------------------------------------------
 .sidebar-menu__button--mobile
-  margin-top: -80px
+  margin-top: -130px
```

Comment:
Even removing the tabs which will be handled in ISAICP-3574, the positioning is not yet ok

![menu_issue](https://user-images.githubusercontent.com/1923476/29129674-e0fbe122-7d27-11e7-9d9f-317137a17322.png)